packages/worker/src/worker.ts

```diff
@@ -563,9 +563,7 @@ export class Worker {
       return bundle;
     } else if (compiledOptions.workflowBundle) {
       if (compiledOptions.bundlerOptions) {
-        throw new ValueError(
-          `You cannot set both WorkerOptions.workflowBundle and .bundlerOptions: if you're providing the bundle, bundlerOptions are not used.`
-        );
+        throw new ValueError(`You cannot set both WorkerOptions.workflowBundle and .bundlerOptions`);
```

Comment:
We should add a lint rule to disallow string templates with no args, definitely not blocking this PR

Reply:
I also use when there's a `'`